analyzers/src/SonarAnalyzer.Common/Rules/ParameterNameMatchesOriginalBase.cs

```diff
@@ -79,5 +79,21 @@ private void VerifyParameters(SyntaxNodeAnalysisContext context, TMethodDeclarat
                 context.ReportDiagnosticWhenActive(Diagnostic.Create(rule, item.actual.GetLocation(), item.actual.ValueText, item.expected.Name, expectedLocation));
             }
         }
+
+        private void VerifyGenericParameters(SyntaxNodeAnalysisContext context, TMethodDeclarationSyntax methodSyntax, IList<IParameterSymbol> actualParameters,
+            IList<IParameterSymbol> expectedParameters, string expectedLocation)
+        {
+            var parameters = ParameterIdentifiers(methodSyntax).ToList();
+            for (var i = 0; i < parameters.Count; i++)
+            {
+                var parameter = parameters[i];
+                var expectedParameter = expectedParameters[i];
+                if (!parameter.ValueText.Equals(expectedParameter.Name, Language.NameComparison)
```

Comment:
This line LGTM. It's usual (and desired) to rename `value As Integer` from interface to `Value As Integer` as VB.NET doesn't suffer from the need to do a variables lower case prefixed.